docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -70,115 +72,167 @@ ifdef::qs[]
-Choose a unique name for your topic. Leave the partitions and retention settings to the default settings.
-
-. Create a service account for the connectors. 
-.. In the *Service Accounts* page of the web console, click *Create service account* to create a new service account.
-.. Enter a unique service account name add an optional description, and click *Create*.
-.. Copy the generated *Client ID* and *Client Secret* to a secure location. These are the credentials that you'll use to connect to this Kafka instance.
-
-. Set the appropriate level of access for the new service account in the Access Control List (ACL) of the Kafka instance.
-.. In the *Kafka Instances* page of the web console, click the name of the Kafka instance that you want the service account to access.
-.. Click the *Access* tab to view the current ACL for this instance.
-.. Click *Manage access*, use the *Account* drop-down menu to select the service account that you previously created, and click *Next*.
-.. Under *Assign Permissions*, use the drop-down menu to select the *Consume from a topic* and the *Produce to a topic* permission options, and set all resource identifiers and values to `Is *`. 
+. Create a topic for connectors:
+.. In the {service-url-connectors}[^] web console, select *Streams for Apache Kafka* > *Kafka Instances* and then click the name of the Kafka instance that you want to add a topic to.
+.. Select the *Topics* tab, and then click *Create topic*.
+.. Type a unique name for your topic. For the quick start example, type *test-topic* for the *Topic Name*. Accept the default settings for partitions and message retention, and replicas.
+. Create a service account for connectors: 
+.. In the web console, select *Service Accounts*, and then click *Create service account*.
+.. Type a unique service account name (for example, *test-service-acct* ) and then click *Create*.
+.. Copy the generated *Client ID* and *Client Secret* to a secure location. You'll use these credentials to configure connections to this service account.
+.. Select the *I have copied the client ID and secret* option, and then click *Close*.
+
+. Set the level of access for your new service account in the Access Control List (ACL) of the Kafka instance:
+.. Select *Streams for Apache Kafka* > *Kafka Instances*, click the name of the Kafka instance that you want the service account to access.
+.. Click the *Access* tab to view the current ACL for this instance and then click *Manage access*.
+.. From the *Account* drop-down menu, select the service account that you previously created, and then click *Next*.
+.. Under *Assign Permissions*, use the drop-down menu to select the *Consume from a topic* and the *Produce to a topic* permission options, and then set all resource identifiers and values to `is "*"`. 
++
+The `is "*"` settings mean that the service account has rights to produce and receive messages for the Kafka topic that you created for Connectors. 
 
 .Verification
```

Comment:
This is causing the "Verification" title to show up in the non-quickstart version. I'd suggest re-wording the qs verification to be something more tangible that the user can verify, and adding a "ifndef::qs[]" for the non-quickstart version.

For example, instead of "Have you completed these steps?", maybe do something like this:

```
.Verification
ifdef::qs[]
* Did you create a topic for connectors?
* Did you create a service account and save the credentials to a secure location?
* Did you set the *Consume from a topic* and *Produce to a topic* permissions for the service account?
endif::[]
ifndef::qs[]:
* Verify that the topic you created exists on the "Topics" page (or whatever the page is called).
* Verify that the service account credentials are copied to a secure location.
* Verify that the *Consume from a topic* and *Produce to a topic* permissions are set for the service account.
endif::[]
```
Alternatively, you could just not include any verification for this procedure.